.storybook/components/Docs/Documentation.tsx

```diff
@@ -1,4 +1,6 @@
 import React, { ReactNode, useEffect } from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore prism.js must be in JS
```

Comment:
@jeremiah-clothier has me convinced that we should always use `@ts-expect-error`. That way we never end up with unnecessary ts comments (since it will error if there is **_no_** error).

It's also nice because for some reason the lint rule doesn't complain about ts-expect-error, as long as there's a comment following is (e.g. the `prism.js must be in JS` here).

Reply:
sgtm!